.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -29,9 +30,14 @@ Please check if your PR fulfills the following requirements:
 - [ ] Docs have been added/updated which fit [documentation template](https://github.com/nventive/Uno/blob/master/doc/.feature-template.md). (for bug fixes / features)
 - [ ] Tests for the changes have been added (for bug fixes / features) (if applicable)
 - [ ] Contains **NO** breaking changes
+- [ ] Associated with an issue (GitHub or internal)
```

Comment:
Let's add a checkmark for "[ ] Updated the [Release Notes](--link-to-_ReleaseNotes.md)"

Reply:
Yes good idea ;) I will add it

Reply:
Done @MatFillion , I will wait for the feedback for this PR and will update the templates for the others repo after :)